### PR TITLE
Core/Fixes: Update Informations

### PR DESCRIPTION
**Pull request by Bodeguero**
* Update Welcome Message

### CODE REVIEW

File: src/server/authserver/Main.cpp [hunk 1]
```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
 
     sLog->outString("%s (authserver)", _FULLVERSION);
     sLog->outString("<Ctrl-C> to stop.\n");
-    sLog->outString("Using configuration file %s.", configFile);
```
Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: why did you remove this line?

Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth I did not remove the line it is in the code:
[Look carefully] (https://github.com/AzerothCore/azerothcore-wotlk/pull/23/commits/61227171a6d47bce1539128b7f89d51d88ac89db#diff-cbc763271de699b2f62e55b0d2755bb5R134)

File: src/server/authserver/Main.cpp [hunk 2]
```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
...
+	sLog->outString(" ");
+	sLog->outString("   █████╗ ███████╗███████╗██████╗  ██████╗ ████████╗██╗  ██╗");           
+	sLog->outString("  ██╔══██╗╚══███╔╝██╔════╝██╔══██╗██╔═══██╗╚══██╔══╝██║  ██║");           
+	sLog->outString("  ███████║  ███╔╝ █████╗  ██████╔╝██║   ██║   ██║   ███████║");           
+	sLog->outString("  ██╔══██║ ███╔╝  ██╔══╝  ██╔══██╗██║   ██║   ██║   ██╔══██║");           
+	sLog->outString("  ██║  ██║███████╗███████╗██║  ██║╚██████╔╝   ██║   ██║  ██║");           
+	sLog->outString("  ╚═╝  ╚═╝╚══════╝╚══════╝╚═╝  ╚═╝ ╚═════╝    ╚═╝   ╚═╝  ╚═╝");           
+	sLog->outString("   ");
```
Comment (DevRival) [inline src/server/authserver/Main.cpp]: Fix tabs :(

File: src/server/authserver/Main.cpp (hunk 1 above)
Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: you're right, the problem is that you changed the code indentation... can you fix it (in all new lines)?

Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth Fix what? What mistakes this time? The code was infuncional? You have errors in time to make the compilation? What is the problem this time in the code?

File: src/server/authserver/Main.cpp (hunk 2 above)
Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @DevRival Fix what? What mistakes this time? The code was infuncional? You have errors in time to make the compilation? What is the problem this time in the code?

Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: the intendation (the code is not well aligned)

image

Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth I'm not seeing anything wrong there and if you look at the code will see that it does not have these spaces and this standard do not know why GitHub shows these spaces because in my local notebook the code is perfect see for yourself below:
[AuthServer_Main.zip](https://github.com/AzerothCore/azerothcore-wotlk/files/380298/AuthServer_Main.zip)

Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: :+1:  no problem my friend, I will fix it for you :)

Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth Thank you for me to understand but if you see the original file this in my notebook does not have these spaces!

Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: which editor do you use?

Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth I use [Notepad ++](https://notepad-plus-plus.org/) Could you give me a better editor?

**Comment by DevRival** [comment]
Quite simply, in the studio there are indentation settings. Error is not here, just standard C++.

File: src/server/authserver/Main.cpp (hunk 2 above)
Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: @Bodeguero right now I'm trying the new Visual Studio Code and it looks nice: https://code.visualstudio.com

another thing: can you give me your github email? so I add you as author of the commit

**Comment by ShinDarth** [comment]
This PR can be closed, see https://github.com/AzerothCore/azerothcore-wotlk/pull/32

File: src/server/authserver/Main.cpp (hunk 2 above)
Comment (Bodeguero) [inline src/server/authserver/Main.cpp]: @ShinDarth Thanks for tips!

Comment (ShinDarth) [inline src/server/authserver/Main.cpp]: you are welcome! and thank you for your contribution
